docs/combine.rst

```diff
@@ -45,29 +45,32 @@ Options
 
         ``all``:   all of the types listed here.
 
-        ``ammo``: AMMO
+        ``ammo``: AMMO. Max 25.
 
-        ``drink``: DRINK
+        ``drink``: DRINK. No Max.
 
-        ``fat``:   GLOB and CHEESE
+        ``fat``:   GLOB and CHEESE. Max 5.
 
-        ``fish``:  FISH, FISH_RAW and EGG
+        ``fish``:  FISH, FISH_RAW and EGG. Max 5.
 
-        ``food``:  FOOD
+        ``food``:  FOOD. Max 20.
 
-        ``meat``:  MEAT
+        ``meat``:  MEAT. Max 5.
 
-        ``parts``: CORPSEPIECE
+        ``parts``: CORPSEPIECE. Max 1.
 
-        ``plant``: PLANT and PLANT_GROWTH
+        ``plant``: PLANT and PLANT_GROWTH. Max 5.
 
-        ``powders``: POWDERS_MISC
+        ``powders``: POWDERS_MISC. Max 10.
 
-        ``seed``: SEEDS
+        ``seed``: SEEDS. Max 1.
```

Comment:
This can be 5 if we filter for only non-plantable seeds. Also, we really should filter for non-plantable seeds or else `-u` could destroy farming